pipeline/tests/test_model.py

```diff
@@ -0,0 +1,157 @@
+"""
+Tests for the pipeline model
+"""
+
+import os
+import unittest
+import psycopg2
+from model.database import update_bounded_similarity_scores, update_popularity_scores, update_trending_scores, package_table_postprocessing
+
+class TestModel(unittest.TestCase):
+    """
+    Tests for the ML pipeline model
+    """
+
+    def setUp(self):
+
+        # Application data
+        app_data = [
+            1,
+            2
+        ]
+
+        # Package data
+        package_data = [
+            {
+                'id': 1,
+                'name': '\'pkg:npm/countup.js@2\'',
+                'monthly_downloads_last_month': 400451,
+                'monthly_downloads_a_year_ago': 0,
+                'categories': 'null',
+                'modified': '\'2019-03-14 12:42:34.846-07\'',
+                'retrieved': '\'2020-04-25 19:03:37.409069-07\'',
+                'app_ids': app_data[-1:]
+            },
```

Comment:
We probably need to create more models for pypi and condo in the end. It should be painful. LOL

Reply:
Agreed. More tests would be good.